packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Event Hooks with ngrok
+excerpt: How to demonstrate Event Hooks using ngrok to expose a local app to the internet.
+layout: Guides
+---
+
+<StackSelector class="cleaner-selector"/>
+
+This guide provides a working example of an Okta Event Hook. It uses the ngrok utility to expose a local application to the internet and receive and respond to Event Hook calls. Event Hooks are outbound calls from Okta that can notify your own software systems of events occurring in your Okta org. The ngrok utility enables the testing of Okta Event Hooks locally, rather than implementing an internet-based production or test external service.
+
+---
+
+**Learning outcomes**
+
+* Understand the Okta Event Hook calls and responses
```

Comment:
Should these have periods?

Reply:
Good question. Do you consider this a full sentence? The subject is implied. If so, yes; if not, no. According to our infodev style guide. Of course, all current versions are inconsistent on use. I'll ping Christine.

Reply:
Confirmed yes. And done. Thanks.